src/RewardsStreamerMP.sol

```diff
@@ -492,10 +478,10 @@ contract RewardsStreamerMP is
 
         VaultData storage vault = vaultData[vaultAddress];
 
-        uint256 accountWeight = vault.stakedBalance + _mpBalanceOf(vaultAddress);
```

Comment:
I'm probably missing something here, but AFAIR we still want to include the bonus MP (for locking) in the reward calculation, not just the staked balance.. no?

Reply:
@0x-r4bbit yes this is just the first PR to remove the previous weights and I'm working to the second one to add to inital + restaked/compound MPs

Reply:
Wonder if it makes sense to do it in one PR then.
But just do your thing and then we check it.